packages/brick_supabase/lib/src/query_supabase_transformer.dart

```diff
@@ -188,7 +188,11 @@ class QuerySupabaseTransformer<_Model extends SupabaseModel> {
 
     return [
       {
-        queryKey: '${_compareToSearchParam(condition.compare)}.${condition.value}',
+        queryKey: condition.compare == Compare.inIterable
+            ? (condition.value is Iterable && (condition.value as Iterable).isNotEmpty
+                ? 'in.(${(condition.value as Iterable).join(',')})'
+                : 'in.()')
+            : '${_compareToSearchParam(condition.compare)}.${condition.value}',
```

Comment:
The nested ternary isn't necessary because the `.join` will produce an empty `()` anyway.
```suggestion
        queryKey: condition.compare == Compare.inIterable && condition.value is Iterable
            ? 'in.(${(condition.value as Iterable).join(',')})'
            : '${_compareToSearchParam(condition.compare)}.${condition.value}',
```

Reply:
@richard457 this wasn't addressed - you don't need the nested ternary